docs/client/concepts/user.mdx

```diff
@@ -0,0 +1,27 @@
+| User ID           | ID representing a unique user. This ID will be used to guarantee consistency of targeting for Feature Gates and Experiments and will be used to evaluate experiment results. If `User ID` is not provided, a `Stable ID` persisted locally will be used for evaluations. | `your_user_id`                                                                                                             |
+| Email             | Email of the user                                                                                                                                                                                                                                                        | `marcos@statsig.com`                                                                                                       |
+| User Agent        | User agent of the browser. This will be decoded to determine the Browser and Operating System of the user's context                                                                                                                                                      | `Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.40 Safari/537.36` |
+| Country           | 2 letter coutry code user                                                                                                                                                                                                                                                | `US`                                                                                                                       |
+| Locale            | Locale of the user                                                                                                                                                                                                                                                       | `en_US`                                                                                                                    |
+| App Version       | Version of the app the user is using                                                                                                                                                                                                                                     | `1.0.1`                                                                                                                    |
+| Custom            | Dictionary that can contain key/value pairs that can be used for Feature Gate targeting. The content of this dictionary will be stored and available after targeting.                                                                                                    | `{is_new_user: true, level: 2}`                                                                                            |
```

Comment:
Only string comparison conditions are possible in console right now, so maybe dont show examples with boolean and int?